server/routes.js

```diff
@@ -26,39 +22,6 @@ module.exports = (expressApp, nextApp) => {
     return res.sendFile(path.join(__dirname, '../public/static/images/favicon.ico.png'));
   });
 
-  /* Helper to enable downloading files that are on S3 since Chrome and Firefox does
-   not allow cross-origin downloads when using the download attribute on an anchor tag,
-   see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download. */
-  app.get('/api/download-file', downloadFileHandler);
```

Comment:
What's happening to the downloadFileHandler?

Reply:
Handled by the `pages/api/download-file` route that is still there, this extra rule no longer needed when removing the `/api/*` proxy in the custom server